[QOLDEV-933] enhance parameter insertion

ThrawnCA opened the pull request:
- Allow specifying default values if parameter could not be loaded from SSM
- Allow `{{ssm: ... }}` syntax as an alternative to `${ssm: ... }`

duttonw committed:
Merge pull request #18 from qld-gov-au/develop

cicd fix up's, setup.py increment

ThrawnCA committed:
Merge pull request #24 from qld-gov-au/develop

Develop to master - enable CKAN 2.10 testing

ThrawnCA committed:
Merge pull request #25 from qld-gov-au/develop

Update GitHub action versions to fix build

ThrawnCA committed:
[QOLDEV-933] enhance parameter insertion

- Allow specifying default values if parameter could not be loaded from SSM
- Allow {{ssm: ... }} syntax as an alternative to

duttonw reviewed (approved):
test please. (at least the regex part)

ThrawnCA committed:
[QOLDEV-933] sync tests with other repos

- Add shell script to handle different Docker Compose versions
- Retrieve logs from GitHub workflow instead of baking it into the test scripts
- Update GitHub Actions
- Don't use a variable in the Docker Compose environment as it doesn't work
- Add health checks to local Docker container startup

ThrawnCA committed:
[QOLDEV-733] replace deprecated 'warn' function with 'warning'

ThrawnCA committed:
[QOLDEV-733] add unit tests for parameter injection

duttonw reviewed (commented):
can we also get a readme update on what it now picks up in i.e. would ${ssm:/missing/value?'default'}
give the default back or just a blank string?

ThrawnCA committed:
[QOLDEV-733] update README with new syntax
